db/migrate/20191022164731_create_service_plans.rb

```diff
@@ -0,0 +1,30 @@
+class CreateServicePlans < ActiveRecord::Migration[5.2]
+  def up
+    create_table :service_plans do |t|
+      t.jsonb :base
+      t.jsonb :modified
+
+      t.bigint :portfolio_item_id
+      t.bigint :tenant_id
+      t.index :tenant_id
+      t.datetime :discarded_at
+      t.index :discarded_at
+
+      t.timestamps
+    end
+
+    puts <<~EOMESSAGE
+      XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+      After this migration, one needs to run
+      `rake db:migrate:service_plans EMAIL=my_real_email@redhat.com`
```

Comment:
@lindgrenj6 @syncrou 
Why can't the import of the service plans be done on a per tenant basis like we do the seeding or manage the settings. Using the rake task seems suitable only for dev testing. We could have an end point like we do for seeding that can import the service plans in the background

Reply:
I thought the idea was we would only capture the service_plan if we were going to edit the survey.  Until then we always pull the live one from Topology so we get the latest version from Tower.

Reply:
@gmcculloug, @mkanoor you're both correct. We can pull this out.
The plan here was to persist known surveys for existing products, but outside of that there is no reason other than having data to work with, so it isn't needed.  

We'll get this pulled out.